spec/factories/generic_files.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
```

Comment:
Let's not use Sufia's generic file factory which we have no control over and it's not even valid in our code base (linter fails on it). Here's a simpler version of theirs: